src/core/UserPreferences.ts

```diff
@@ -50,4 +51,13 @@ export default class implements UserPreferencesProvider {
   getSpacing(): number {
     return this.#settings.get_int("window-spacing");
   }
+
+  /**
+  * Whether windows should be automatically maximized when resized to fill
+  * the entire screen.
+  * * @returns True if windows should be auto-maximized, false otherwise.
+  */
```

Comment:
```suggestion

  /**
  * @returns Whether to maximize windows when selection fills the entire grid.
  */
```

(try sticking with the formatting)